Marlin/pins_ZRIB_V20.h

```diff
@@ -0,0 +1,20 @@
+  #include "pins_MKS_13.h"
+#include "pins_RAMPS.h"
+  #undef  SERVO1_PIN 6
+  #ifndef ZRIB_V20_D6_PIN
+  #define ZRIB_V20_D6_PIN 6
+  #endif
+  #ifndef ZRIB_V20_D9_PIN
+  #define ZRIB_V20_D9_PIN 9
+  #endif
+
+
+
+  #undef FAN_PIN
+  #undef E0_AUTO_FAN_PIN
+  #undef E1_AUTO_FAN_PIN
+  #undef E2_AUTO_FAN_PIN
+  #define FAN_PIN        ZRIB_V20_D6_PIN 

```

Comment:
To override all general use of the usual `FAN_PIN` (D9) you should use this line instead, before the `#include pins_MKS_13.h` line:
```cpp
#define RAMPS_D9_PIN ZRIB_V20_D6_PIN
```
This allows the pin to be reassigned to a heater or other use according to the configuration.